js/idex.js

```diff
@@ -227,11 +245,9 @@ module.exports = class idex extends Exchange {
             const quoteId = this.safeString (entry, 'quoteAsset');
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
-            const basePrecisionString = this.safeString (entry, 'baseAssetPrecision');
-            const quotePrecisionString = this.safeString (entry, 'quoteAssetPrecision');
-            const basePrecision = this.parseNumber (this.parsePrecision (basePrecisionString));
-            let quotePrecision = this.parseNumber (this.parsePrecision (quotePrecisionString));
-            quotePrecision = this.safeNumber (entry, 'tickSize', quotePrecision);
+            const basePrecision = this.parseNumber (this.parsePrecision (this.safeString (entry, 'baseAssetPrecision')));
+            const quotePrecision = this.parseNumber (this.parsePrecision (this.safeString (entry, 'quoteAssetPrecision')));
+            const pricePrecision = this.safeNumber (entry, 'tickSize');
```

Comment:
I think we've lost the quotePrecision / quoteAssetPrecision here.

Reply:
That was intentional by me. 
quoteAssetPrecision was being set as a backup for `precision.price` (if tick-size was not present in response) which is incorrect. across lib i've found different cases and have posted the question in discord (whenever you have time, please check `limit_price_min` thread), and I strongly think it's definitely incorrect to set `precision.price` from quote-asset-precision, as it pair's price precision (ticksize) has no direct link to quote-asset-precision. 

for example, BTC/USDT - in that market, ticksize might have resolution of 0.01, while quote-asset-precision (of USDT) might be 0.000001 or whatever. so, from lib, we might need to remove any assignments of quote-precision to `precision.price`.

Reply:
I confirm, all of the markets have tickSize property.